tests/interpretation/test_linear_probe.py

```diff
@@ -10,27 +10,26 @@
 
 import neurox.interpretation.linear_probe as linear_probe
 
+
```

Comment:
Unfortunately because of bad instructions, you ended up formatting this very poorly formatted file; Sorry for that! Do you mind pushing your changes pre-formatting so the diff is easier to review?

Reply:
I just noticed that the formatting is in a nice clean commit, so it should be as easy as `git revert <formatting-commit-id>` and pushing!